modules/standard/DateTime.chpl

```diff
@@ -577,7 +584,7 @@ module DateTime {
       ret = ret + "." + makeNDigits(6, microsecond);
     }
     var offset = utcoffset();
-    if tzinfo != nil {
+    if tzinfo.borrow() != nil {
```

Comment:
Do you think it'd be worthwhile having a `proc isEmpty` or something on Shared?

Reply:
Actually it might be even slicker if we made support for `==` and `!=` on Owned/Shared combined with a nil type.

Reply:
Yes, `==` and `!=` for Owned/Shared with `nil` would be nice.